source/nodejs/adaptivecards-designer/src/containers/host-container.ts

```diff
@@ -15,6 +16,40 @@ export abstract class HostContainer {
 
     abstract renderTo(hostElement: HTMLElement);
 
+    private static playNextTTS(output: any[], iCurrent: number) {
```

Comment:
There is no "Read this card" button in the Designer's UI, therefore this isn't necessary since it's only there to support the implementation of card narration.

Reply:
Don't change - we'll add the "read this card" button later ourselves, so code can stay as is